File: src/Plus One/Plus One.cpp
class Solution {
	public:
		vector<int> plusOne(vector<int> &digits) 
		{
			vector<int> ret ( digits.begin(), digits.end() );
			int len = ret.size();
			ret[len-1]++;

			for(int i = len-1 ; i > 0 ; i--)
			{   
				ret[i-1] += ret[i] / 10; 
				ret[i] %= 10; 
			}   

			if(ret[0] == 10) 
			{   
				ret[0] = 1;
				ret.push_back(0);
			}   

			return ret;
		}
};

File: src/Plus One/Plus One.py
class Solution:
	# @param digits, a list of integer digits
	# @return a list of integer digits
	def plusOne(self, digits):
		num = 0 
		for i in digits:
			num = 10 * num + i 
		return [int(i) for i in str(num+1)]
